### PR TITLE
added doc for tolerations

### PR DESCRIPTION
**Pull request by tikr7**

### CODE REVIEW

File: notebooks/applications/spark/README.md [hunk 1]
```diff
@@ -154,3 +191,8 @@ AttributeError: 'ScalaMonitor' object has no attribute 'comm'
 ```
 
 Solution: Shutdown the kernel and close the notebook. Re-opening the notebook and running the cells should fix it.
+
+
+## Culling (auto suspend after long idled notebooks)
+
+Due to costs savings, all notebooks get into supend mode after the kernel is idle for more than 6 hours. You can simply start your notebook within the UI again if needed. If you start your "Feierabend" or weekend, please shut them down manually and do not wait for the culling feature. Our budgets is limited and it is better to use it for calculation and training rather than for idled notebooks.
```
Comment (ncbkr) [inline notebooks/applications/spark/README.md:198]: typo: "suspend mode"

File: notebooks/configuration.md [hunk 2]
```diff
@@ -141,3 +141,5 @@ spot)
     type is suitable for any kind of workload that are safe to interrupt.
     Data which is stored on Persistent Storage will not be lost. Spot
     instances are ideal for performing pipeline tasks.
+
+-   Big Notebooks: Under `Tolerations Group` select `Big Notebooks` if you need a lot CPU/RAM. (Due to cost savings it is not possible to use those without the `Tolerations Group` anymore.)
```
Comment (ncbkr) [inline notebooks/configuration.md:145]: I don't think the addition in brackets is required ;)

**Review by ncbkr — APPROVED** [review]
Please add a short description to your pull requests.

Had some small comments but looks good to me

File: notebooks/configuration.md (hunk 2 above)
Comment (tikr7) [reply notebooks/configuration.md]: fixed

File: notebooks/applications/spark/README.md (hunk 1 above)
Comment (tikr7) [reply notebooks/applications/spark/README.md]: fixed

**Pull request merged**